### ISSUE TITLE
See the code in OM#

### ISSUE THREAD

**Issue by charlesneimog** [cac-t-u-s/om-sharp-users#23]
Hello Jean, 

Is it possible to see the code of om# objects in one easy way (like press E in om6).

**Comment by j-bresson** [comment]
Yes, you can find this in the "Help" menu: "Find Source" (shortcut is `cmd`/`ctrl`+`shift`+`E`)

**Comment by charlesneimog** [comment]
Hi Jean, I was trying to test the function Find Source but I am receiving this message, "no definition found for arithm-ser", "no definition found for lin->db", etc. 

Just reporting! I tested it on Windows and Debian with the version 1.2. 

Thank you!

**Comment by j-bresson** [comment]
Thanks for reporting. I think I spotted a bug in the release packaging, preventing this feature to work.
I think it will be ok in v1.3

**Comment by j-bresson** [comment]
Let me know if it works (Linux shoudl be coming soon) https://github.com/cac-t-u-s/om-sharp/releases/tag/v1.3

**Comment by charlesneimog** [comment]
On Windows it is working now. Thanks!!!
